node-api/handlers/beacon/genesis.go

```diff
@@ -35,11 +37,23 @@ func (h *Handler) GetGenesis(_ handlers.Context) (any, error) {
 	if len(genesisRoot) == 0 {
 		return nil, types.ErrNotFound
 	}
+
+	genesisForkVersion, err := h.backend.GenesisForkVersion(utils.Genesis)
+	if err != nil {
+		return nil, err
+	}
+	genesisForkVersionHex := "0x" + hex.EncodeToString(genesisForkVersion[:])
+
```

Comment:
I wonder if we create a canonical way to represent fork versions? We should check what ethereum does
cc @calbera

Reply:
From the spec they have returned
```
     "genesis_fork_version": "0x00000000"
```

Reply:
For us, similarly we should return `"0x04000000"`

Reply:
Yes, this is the response for us `"genesis_fork_version": "0x04000000"`